### PR TITLE
[xxx] Fix ETP issues

### PR DESCRIPTION
**Pull request by darokel**
### Context

Fixes https://sentry.io/organizations/dfe-teacher-services/issues/3678466933/?referrer=slack and a few issues with ETP

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -134,7 +134,7 @@
     t.string "additional_gcse_equivalencies"
     t.boolean "can_sponsor_skilled_worker_visa", default: false
     t.boolean "can_sponsor_student_visa", default: false
-    t.string "campaign_name", default: "no_campaign"
+    t.integer "campaign_name"
```
Comment (defong) [inline db/schema.rb:137]: Don't we need another migration to drop column and create column as the data type has changed?

**Review by goodviber — COMMENTED** [review]
The deploy app is still not working correctly

File: db/schema.rb (hunk 1 above)
Comment (darokel) [reply db/schema.rb]: done

**Comment by darokel** [comment]
@goodviber should be fixed now, have a look

**Review by goodviber — APPROVED** [review]

**Pull request merged**